src/strategies/terraform-module.ts

```diff
@@ -88,6 +89,26 @@ export class TerraformModule extends BaseStrategy {
         }),
       });
     });
+
+    // Update metadata.yaml and metadata.display.yaml to current candidate version.
+    const metadataFiles = await Promise.all([
```

Comment:
I think this could be simplified to:

```js
const metadataFiles = await this.githubFilesByFilename(...)
```

You shouldn't need the `Promise.all` unless you are chaining together multiple async requests.

Reply:
Done.